### ISSUE TITLE
Principles contains an example with explanatory of hazards consequences for user which is not addressed by the Technics

### ISSUE THREAD

**Issue by gautierchomel** [w3c/publ-a11y#390]
The Principles Hazards descriptive example contains an explanatory sentence [... which can cause discomfort, distraction, photosensitive seizures, or motion sickness.](https://github.com/w3c/publ-a11y/blob/e9b200e81c5a5363a0418bfe0bc09cc08ecca516/UX-Guide-Metadata/draft/principles/index.html#L715) for which no guidance is given in the technics. 

This explanation is essential because it's a user-centric approach to understanding what hazards mean (i.e. detailing the effect instead of the trigger)

**Comment by gregoriopellegrino** [comment]
In the principles we use the compact version, right?

**Comment by gautierchomel** [comment]
Yes, but in that case, the descriptive example presented in the principles cannot be achieved with the techniques.

**Comment by gregoriopellegrino** [comment]
Riusciamo a ristrutturare gli esempi dei principles in modo che si possano adattare a come abbiamo impostato le techniques?

Like:
`Rapidly changing lights or visual stimuli, simulated movements, which can cause discomfort, distraction, photosensitive seizures, or motion sickness.`

**Comment by gautierchomel** [comment]
To me, the techniques serve the principles. So, we should adapt the techniques, not the principles.

**Comment by gregoriopellegrino** [comment]
I think that even in localizations in other languages the sentence structure does not work. It would be best if for each hazard there is only one string to localize, whether it is compact or descriptive, and it always appears in the same position in the sentence construction.

Right now the techniques are agnostic as to whether the implementers are using it to generate the descriptive or compact version, and I think this is a good feature.

**Comment by gregoriopellegrino** [comment]
@GeorgeKerscher we should fix the second example both descriptive explanations and compact explanations to have the same structure

**Comment by GeorgeKerscher** [comment]
I believe we have addressed this in the new version of the guidelines. I will close. If there is still a problem, please create a new issue with the details.
